groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/typinghooks/GroovyTypedBreakInterceptor.java

```diff
@@ -163,6 +163,9 @@ public void insert(MutableContext context) throws BadLocationException {
         }
 
         if (id == GroovyTokenId.STRING_LITERAL || id == GroovyTokenId.STRING_CH && offset < ts.offset()+ts.token().length()) {
+            if (id == GroovyTokenId.STRING_LITERAL && ts.offsetToken().text().toString().startsWith("\"\"\"")) {
```

Comment:
Good! Could you add ```"'''"``` check as well?

Reply:
Thank you for noticing. I completely forgot that Groovy has strings prefixed with '